kube_scheduler/manifest.json

```
@@ -0,0 +1,24 @@
+{
+  "display_name": "Kube_scheduler",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "kube_scheduler",
+  "metric_prefix": "kube_scheduler.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Monitors the Kubernetes Scheduler",
+  "guid": "ec7c029f-86c2-4202-9368-1904998a646c",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Kube_scheduler Integration",
+  "categories": [
+    ""
```

```suggestion
    "containers"
```

There is at least the `containers` category that can be applied here, you may find the complete list of categories here: https://docs.datadoghq.com/integrations/

---

I will add "containers" and "orchestration"